lesson5/public/index.html

```diff
@@ -0,0 +1,22 @@
+<html lang = "en">
+  <head>
+    <meta charset = "UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name: <input type = 'text' id = 'nameBox'/>
+    Comment: <input type = 'text' id = 'commentBox'/>
+    <input type = 'submit' id = 'submitBox'>
+    <script>
+const button = document.getElementById('submitBox');
+button.onclick =()=>{
+  const name = document.getElementById('nameBox').value;
+  const comment = document.getElementById('commentBox').value;
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('GET',`/XMLHttpRequest?nameBox=${name}&commentBox=${comment}`);
```

Comment:
nameBox and commentBox better changed to nameBoxValue or something. They don't have to match the id name, actually it's better that they don't